modules/apps/foundation/organizations/organizations-api/src/main/resources/content/Language.properties

```diff
@@ -0,0 +1,4 @@
+organization-link-configuration-name=Organization Links
+organization-link-enabled-description=Check to enable defining arbitrary links between organizations.
+organization-link-types=Organization Link Types
+organization-link-types-description=The name of an Organization Link type
```

Comment:
I think the description message should be more descriptive, and it should be a complete sentence.